align/compiler/create_database.py

```diff
@@ -71,10 +71,8 @@ def propagate_const_top_to_bottom(self, top_name, traversed):
         if not all_subckt:
             return
         for const in top.constraints:
-            global_const = [constraint.IsDigital, constraint.AutoConstraint,
-                            constraint.FixSourceDrain, constraint.DoNotUseLib,
-                            constraint.KeepDummyHierarchies, constraint.MergeSeriesDevices,
-                            constraint.MergeParallelDevices, constraint.IdentifyArray,
+            global_const = [constraint.ConfigureCompiler,
+                            constraint.DoNotUseLib,
```

Comment:
`DoNotUseLib` could also be rolled under `ConfigureCompilier` as `ExcludeTemplates` ?

Reply:
Here I only merged switches with bool options. DoNotUseLib, DoNotIdentify, ClkPorts both are compiler configuration handles but are kept as separate constraints. Do we want to rename them with a compiler tag?

Reply:
We can do this in separate PR if this is the only concern?

Reply:
Sure, let's do this in a future PR then.